packages/chart/src/components/LineChart.tsx

```diff
@@ -121,24 +183,35 @@ export default function LineChart({ xScale, yScale, getXAxisData, getYAxisData,
                 strokeOpacity={1}
                 shapeRendering='geometricPrecision'
                 strokeDasharray={lineType ? handleLineType(lineType) : 0}
+                // this is needed to get the non-numerics to not break the chart
+                // - but if you add isNumber here and remove cleanedData above
+                // it will fail
                 defined={(item, i) => {
-                  return item[config.runtime.seriesLabels[seriesKey]] !== '' && item[config.runtime.seriesLabels[seriesKey]] !== null
+                  return item[config.runtime.seriesLabels[seriesKey]] !== '' && item[config.runtime.seriesLabels[seriesKey]] !== null && item[config.runtime.seriesLabels[seriesKey]] !== undefined
                 }}
               />
               {config.animate && (
                 <LinePath
                   className='animation'
                   curve={allCurves.curveLinear}
-                  data={data}
+                  data={cleanData(data)}
                   x={d => xScale(getXAxisData(d))}
-                  y={d => (seriesAxis === 'Right' ? yScaleRight(getYAxisData(d, seriesKey)) : yScale(getYAxisData(d, seriesKey)))}
+                  y={d => {
```

Comment:
consider using a handler when longer functions are used:

`const handleLinePath = () => {}`
...
`y={handleLinePath}`

Once ternarys get to a certain point in length, in my opinion it feels better to have them moved into there own handler. I'll let you be the judge here.

Reply:
I admit I was not sure if that even needed adjusting so was iterating on that... in the end that is when I should have pulled that out into a handler.  No problem I can do that.

Reply:
I had done this early on in the testing of where the problem was... cleaning the data prior to sending it into the component fixes the problem and this extra checking here is redundant so removing it.